web-frontend/package.json

```diff
@@ -20,11 +20,10 @@
     "vue-writer": "^2.0.2"
   },
   "devDependencies": {
-    "@vitejs/plugin-vue": "^3.2.0",
```

Comment:
As noted in [docs](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue), since we are using Vue 3.4.31, there is no need to add `@vitejs/plugin-vue` as dependencies.
> Note: as of vue 3.2.13+ and @vitejs/plugin-vue 1.9.0+, @vue/compiler-sfc is no longer required as a peer dependency.